tools/convert-unicode-emoji-data

```diff
@@ -25,24 +25,26 @@ SCRIPT_NAME = PurePath(__file__).name
 emoji_dict = EMOJI_NAME_MAPS
 
 emoji_map = {}
-for emoji_code, emoji_names in emoji_dict.items():
-    emoji_map[emoji_names["canonical_name"]] = emoji_code
-    for emoji in emoji_names["aliases"]:
-        emoji_map[emoji] = emoji_code
+for emoji_code, emoji_data in emoji_dict.items():
+    emoji_map[emoji_data["canonical_name"]] = {
+        "code": emoji_code,
+        "aliases": emoji_data["aliases"],
+    }
 
 ordered_emojis = OrderedDict(sorted(emoji_map.items()))
 
 with OUTPUT_FILE.open("w") as f:
     f.write(
         "from collections import OrderedDict\n\n\n"
+        "# fmt: off\n"
         f"# Generated automatically by tools/{SCRIPT_NAME}\n"
         "# Do not modify.\n\n"
         "EMOJI_DATA = OrderedDict(\n    [\n"
     )
-    for emoji_code, emoji_name in ordered_emojis.items():
-        # {'smile': {'code': '263a'}}
-        f.write(f'        ("{emoji_code}", {{"code": "{emoji_name}"}}),\n')
-    f.write("    ]\n)\n")
+    for emoji_name, emoji in ordered_emojis.items():
+        # {'smile': {'code': '263a', 'aliases': []}}
+        f.write(f'        ("{emoji_name}", {emoji}),\n')
+    f.write("    ]\n)\n" "# fmt: on\n")
```

Comment:
Black combination issue?

Reply:
Yes, those were in different lines earlier but black moved them in one line and didn't combine them.